### ISSUE TITLE
onMouseEnter behaving like onMouseOver

### ISSUE THREAD

**Issue by blup** [dekujs/deku#240]
When I bind onMouseEnter to a container that has children divs, the event is triggered on hovering every child instead of just once. If i bind it manually on afterMount it works fine of course.

```javascript
export function render({props, state}, set) {
  return (
    <div class="container" onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
      <div>Child</div>
      <div>Child</div>
    </div>
  )
}

// vs

export function afterMount({props, state}, el, set) {
   el.addEventListener('mouseenter', onMouseEnter)
   el.addEventListener('mouseleave', onMouseLeave)
}

export function render({props, state}, set) {
  return (
    <div class="container">
      <div>Child</div>
      <div>Child</div>
    </div>
  )
}

```

**Comment by anthonyshort** [comment]
This looks like a bug with the event delegation. In the next version we're not using delegation so it shouldn't be a problem. I'll leave this open so I can remember to add a test case for it.

**Comment by anthonyshort** [comment]
Closing this now as it should be fixed in the latest version, 2.0
